### ISSUE TITLE
Suggestion: Add Subtitles

### ISSUE THREAD

**Issue by dinferizovic** [ceberous/theWatchFlix#1]
I think subtitles would be beneficial for non-native speakers :) 

You could scrape subtitles from sites like OpenSubtitles

**Comment by ceberous** [comment]
Good idea :) There's lots of stuff to improve upon. I'll try and add some of the ones I know about. The problem though is, this was a more of a proof-of-concept. The front end merely is a bare bones interface. Its not stylized at all. My next plan for this was to port it to a C program and then wire some buttons to a raspi. That way I can ship it to my friends as a plug and play device. I think this is the preferred route because a lot of the provider servers are really careful as to who they are allowing a download stream to. This is why if you run this on your own machine you can access the vodlocker links. But when you access it via my domain @ ceberous.com vodlocker blocks the request, presumably because it is aware of digitaloceans ip and believes it could host malicious bots. A normal user would never be located at digitalocean.... so despite many attempts to rewrite the referrer header... I still can't prevent vodlocker from blocking the stream when I try to host this in various places.... digitalocean.....heroku.... the like.

**Comment by hayzamjs** [comment]
Why port it to C, that'll make things a lot more difficult to work with, The underlying API is good. Maybe just focus this project on the API and drop the front-end as a whole!
